lib/srv/db/cloud/watchers/watcher.go

```diff
@@ -41,6 +41,11 @@ type WatcherConfig struct {
 	Interval time.Duration
 }
 
+// IsEmpty returns true if no matchers are found.
```

Comment:
If this code just matches the dbs on the different cloud providers, it should be moved to the discovery service.
For compatibility, the db agent can import it, but I think it makes more sense to move it to the final destination right away.

Reply:
👍 will move